cookbook/lvgl.rst

```diff
@@ -928,12 +928,139 @@ To put a title bar behind the status icon, we need to add it to each page, also
+                        id: but_cov_down_east
+                        width: 70
+                        height: 68
+                        widgets:
+                          - label:
+                              id: cov_down_east
+                              align: center
+                              text: "\U000F0045" # mdi:arrow-down
+
+                    - label:
+                        text: "South"
+                    - btn:
+                        id: but_cov_up_south
+                        width: 70
+                        height: 68
+                        widgets:
+                          - label:
+                              id: cov_up_south
+                              align: center
+                              text: "\U000F005D"
+                    - btn:
+                        id: but_cov_stop_south
+                        width: 70
+                        height: 68
+                        widgets:
+                          - label:
+                              id: cov_stop_south
+                              align: center
+                              text: "\U000F04DB"
+                    - btn:
+                        id: but_cov_down_south
+                        width: 70
+                        height: 68
+                        widgets:
+                          - label:
+                              id: cov_down_south
+                              align: center
+                              text: "\U000F0045"
+
+                    - label:
+                        text: "West"
+                    - btn:
+                        id: but_cov_up_west
+                        width: 70
+                        height: 68
+                        widgets:
+                          - label:
+                              id: cov_up_west
+                              align: center
+                              text: "\U000F005D"
+                    - btn:
+                        id: but_cov_stop_west
+                        width: 70
+                        height: 68
+                        widgets:
+                          - label:
+                              id: cov_stop_west
+                              align: center
+                              text: "\U000F04DB"
+                    - btn:
+                        id: but_cov_down_west
+                        width: 70
+                        height: 68
+                        widgets:
+                          - label:
+                              id: cov_down_west
+                              align: center
+                              text: "\U000F0045"
+
+This saved you from a considerable amount of manual calculation of widget positioning, if you wanted to place each of them manually with ``x`` and ``y``! You only need to decide for a common width of your widgets, and a height which will distribute them the way you like it. (:ref:`lvgl-cook-icontext` below shows how to use custom icons)
```

Comment:
```suggestion
This saved you from a considerable amount of manual calculation of widget positioning which would otherwise be required to place them manually with ``x`` and ``y``! You only need to determine a common width and height for your widgets to distribute them on the page as you prefer. (:ref:`lvgl-cook-icontext` below shows how to use custom icons.)
```